pandapower/test/loadflow/test_results.py

```diff
@@ -685,6 +685,34 @@ def test_bus_bus_switch(result_test_network, v_tol=1e-6, i_tol=1e-6, s_tol=5e-3,
     assert abs(net.res_bus.vm_pu.at[b3] - v2) < v_tol
     assert abs(net.res_bus.vm_pu.at[b2] == net.res_bus.vm_pu.at[b2])
 
+    for col in ("p_from_mw", "p_to_mw", "q_from_mvar", "q_to_mvar", "i_ka", "loading_percent"):
+        assert col in net.res_switch
+
+    assert isnan(net.res_switch.p_from_mw).all()
+
+    # now test with some switches that have non-zero impedance:
+    net.switch.loc[[2,3], "z_ohm"] = 1e-3
+    pp.runpp(net)
+
+    p_from_ref = -0.29816
+    p_to_ref = 0.29816
+    q_from_ref = -0.96063
+    q_to_ref = 0.96063
+
+    assert isclose(net.res_switch.p_from_mw.at[2], p_from_ref, rtol=1e-3, atol=1e-6)
```

Comment:
please set rtol to 0 and only use atol

Reply:
If I use rtol=0, how does it work? There is always a relative tolerance if I use an only approximately accurate value as a reference.

Reply:
i have updated the code and used exact values, i just dont see the point to use isclose when using exact values. I changed the implementation by using exact values and therefore I will resolve this conversation, maybe we can discuss the usage of isclose later.